specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json

```diff
@@ -278,6 +388,68 @@
           }
         }
       }
+    },
+    "/keys/{keyName}/managedidentitymoderestore": {
```

Comment:
Do you have suitable validation in place to make sure no key version could ever be "managedidentitymoderestore"?